deployments/base/cleaner/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: boskos-cleaner
+  labels:
+    app: boskos-cleaner
+  namespace: boskos
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: boskos-cleaner
+  template:
+    metadata:
+      labels:
+        app: boskos-cleaner
+    spec:
+      serviceAccountName: boskos-cleaner
+      terminationGracePeriodSeconds: 300
+      containers:
+        - name: boskos-cleaner
+          image: gcr.io/k8s-staging-boskos/cleaner:v20200617-f289ba6
+          args:
+            - --boskos-url=http://$(BOSKOS_SERVICE).$(BOSKOS_NAMESPACE).svc.cluster.local.
+            - --namespace=$(BOSKOS_NAMESPACE)
```

Comment:
FWIW, we use the controller-based implementation since February 27th and haven't noticed any issue so far: https://github.com/openshift/release/pull/7347

Reply:
Thanks, I've added an additional commit so the changes needed are obvious.